early_tpm.c

```diff
@@ -1002,20 +1002,25 @@ struct tpm *enable_tpm(void)
 
 void tpm_request_locality(struct tpm *t, u8 l)
```

Comment:
This function is declared as void but you are returning a value?

Reply:
Fixed